covid_moonshot_ml/utils.py

```diff
@@ -202,6 +202,8 @@ def train(
         training to ensure the appropriate number of epochs is reached
     train_loss : list[float], default=[]
         List of train losses from previous epochs. Used when restarting training
+    val_loss : list[float], default=[]
```

Comment:
The default is now `None`
```suggestion
    val_loss : list[float], default=None
```